plugins/rtorrent/rtom_allsessions_vol

```diff
@@ -3,7 +3,7 @@
 # xmlrpc based munin plugin for monitoring rtorrent's torrent count
 # prerequisites:
 #  - rtorrent 0.7.5 or newer compiled with --with-xmlrpc-c
-# check http://libtorrent.rakshasa.no/wiki/RTorrentXMLRPCGuide for further information
+# check http://libtorrent.rakshasa.no/wiki/RTorrentXMLRPCGuide for further informations
```

Comment:
I am not a native speaker, but I am quite confident, that there is no plural for "information" in English :)